sycl/test/basic_tests/id.cpp

```diff
@@ -437,6 +437,36 @@ int main() {
           (number_5 == numValue));
   }
 
+  {
```

Comment:
Shouldn't we have this checks even if `__SYCL_DISABLE_ID_TO_INT_CONV__` is defined?

Reply:
Fixed, now the checks runs independently of the macro.